mobify/navitron
# Apply Modern Standards

Reviewers: @mlworks @jansepar @fractaltheory @mikenikles @MikeKlemarewski 
## Changes
- Prep publish to NPM
- Use jQuery and latest Velocity release
- Migrate from Bower to NPM for dependencies
- Migrate JS linting to eslint
- Migrate SCSS compilation to libsass
- Lint SCSS with sass-lint
- Update circleCI config with modern testing steps
- Add some GitHub templates for contributing, issues, and PRs
- Fix circleCI tests (phantomJS does not support requestAnimationFrame)
## Todos:
- [x] +1 (Major version bump)
## How to Test
- Checkout code
- Refresh node modules with `rm -rf node_modules && npm install`
- Test with `grunt test` or `grunt test:browser`
- Check that the examples still work with `grunt examples`
- Check out [this branch of Shop Deluxe](https://github.com/mobify/deluxe-mobile/tree/DNM-testing-modern-navitron) that uses the new Navitron code
